### ISSUE TITLE
Repro for RAFT variants

### ISSUE THREAD

**Issue by SamriddhiJain** [princeton-vl/RAFT-3D#4]
Hi

Thanks for sharing the work. I'm trying to repro the RAFT variants (2D flow backprojected, 3D flow) mentioned in the paper. I'm facing some numerical instabilities and was wondering did you experience anything like that? And if yes, did you try some different learning schedules as of RAFT? Would it be possible to share that code?

Thanks

**Comment by zachteed** [comment]
Hi, I used the same learning rate for all the ablations of the RAFT variants. I don't remember having any issues with numerical instability with these models. Are you having issues with the training of the model included in the repo, or just the variants mentioned in the paper?

I can provide code for the 3D and 2D-backproj in 1-2 days, just need to check that they still work fine with the refactored code

**Comment by SamriddhiJain** [comment]
>  Are you having issues with the training of the model included in the repo, or just the variants mentioned in the paper?
Just the variants right now.

> I can provide code for the 3D and 2D-backproj in 1-2 days, just need to check that they still work fine with the refactored code
Great, thanks.

**Comment by SamriddhiJain** [comment]
Hey, any updates on this?
